### PR TITLE
Revert mapping item details edit section

### PR DESCRIPTION
**Pull request by mariia-aloshyna**

### CODE REVIEW

File: src/settings/MappingProfiles/detailsSections/edit/MappingInstanceDetails.js [hunk 1]
```diff
@@ -101,3 +108,10 @@ export const MappingInstanceDetails = ({
     </AccordionSet>
   );
 };
+
+MappingInstanceDetails.propTypes = {
+  initialFields: PropTypes.shape(mappingInstanceInitialFieldsShape).isRequired,
+  referenceTables: PropTypes.shape(mappingInstanceRefTablesShape).isRequired,
+  setReferenceTables: PropTypes.func.isRequired,
+  okapi: PropTypes.shape(okapiShape).isRequired,
```
Comment (VictorSoroka1) [inline src/settings/MappingProfiles/detailsSections/edit/MappingInstanceDetails.js:116]: What do you think about refactoring to be able to use it in the following way? As I see this how we use `stripesShape`.
```suggestion
  okapi: okapiShape.isRequired,
```

File: src/settings/MappingProfiles/detailsSections/edit/ItemDetailSection/ElectronicAccess.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  TextField, RepeatableField,
```
Comment (Vladyslav-Velytskyi) [inline src/settings/MappingProfiles/detailsSections/edit/ItemDetailSection/ElectronicAccess.js:10]: Please, split this values in separate lines

File: src/settings/MappingProfiles/detailsSections/edit/ItemDetailSection/ItemNotes.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  TextField, RepeatableField,
```
Comment (Vladyslav-Velytskyi) [inline src/settings/MappingProfiles/detailsSections/edit/ItemDetailSection/ItemNotes.js:10]: Please, split this values in separate lines

File: src/settings/MappingProfiles/detailsSections/edit/ItemDetailSection/LoanAndAvailability.js [hunk 4]
```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  TextField, RepeatableField,
```
Comment (Vladyslav-Velytskyi) [inline src/settings/MappingProfiles/detailsSections/edit/ItemDetailSection/LoanAndAvailability.js:10]: Same here

File: src/settings/MappingProfiles/detailsSections/edit/MappingInstanceDetails.js (hunk 1 above)
Comment (mariia-aloshyna) [reply src/settings/MappingProfiles/detailsSections/edit/MappingInstanceDetails.js]: Good idea, thanks!
Done

**Pull request merged**